vignettes/col_formatting.Rmd

```diff
@@ -14,6 +14,14 @@ knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>"
 )
+
+suggested_dependent_pkgs <- c("dplyr")
+knitr::opts_chunk$set(
+  eval = all(vapply(suggested_dependent_pkgs, (requireNamespace),
+    logical(1),
+    quietly = TRUE
+  ))
+)
```

Comment:
- why in brackets? `(requireNamespace)`
- let's try to merge it with the previous options setter